### PR TITLE
Use new Page and PageContent components on site

### PR DESCRIPTION
**Pull request by taysea**
[Deploy Preview](https://deploy-preview-2432--keen-mayer-a86c8b.netlify.app/components/button)

#### What does this PR do?

Updates DS site layout to use the new Page and PageContent components. Adds necessary theme overrides to keep horizontal padding consistent with current layout.

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

### CODE REVIEW

**Comment by ericsoderberghp** [comment]
Can you provide a little more detail on?
> Adds necessary theme overrides to keep horizontal padding consistent with current layout.

I was hoping that the theme would work as-is.

**Comment by jcfilben** [comment]
> I was hoping that the theme would work as-is.

I was expecting we would need to change the `min` width for the hpe theme (see [relevant discussion](https://github.com/grommet/grommet/pull/5960#discussion_r810356646)) as well as adding horizontal for the 'xsmall' and 'xlarge' breakpoints. However I wasn't expecting we would need to override the horizontal padding for the 'small', 'medium', and 'large' breakpoints

**Comment by jcfilben** [comment]
Looks like we need extra padding under the roadmap card on the [extend page](https://deploy-preview-2432--keen-mayer-a86c8b.netlify.app/extend)

Screen Shot 2022-03-08 at 10 50 24 AM

**Comment by taysea** [comment]
> However I wasn't expecting we would need to override the horizontal padding for the 'small', 'medium', and 'large' breakpoints

Good comments @jcfilben -- this is a discussion point I want to bring up with the team. If we stick with the paddings we want for app page layouts, it makes the sites layout much wider (not necessarily a bad thing, but a change). I will post something in staff channel.

File: aries-site/src/themes/aries.js [hunk 1]
```diff
@@ -51,6 +51,30 @@ export const aries = deepMerge(hpe, {
...
+      },
+      medium: {
+        pad: { horizontal: 'large' },
+      },
+      large: {
+        pad: { horizontal: 'xlarge' },
+      },
+      xlarge: {
+        pad: { horizontal: 'xlarge' },
+      },
```
Comment (halocline) [inline aries-site/src/themes/aries.js:75]: Are these nested too deep? They are currently a child of 'wide'. Shouldn't they be a child of 'page'?

Comment (jcfilben) [reply aries-site/src/themes/aries.js:75]: This is aligned with the current theme structure of Page. Padding is defined within page[kind][size] in addition to any other box props.

File: aries-site/src/pages/index.js [hunk 2]
```diff
@@ -88,7 +83,7 @@ const Index = () => {
       <Meta title={title} description={pageDetails.seoDescription} />
       <Box>
         <Intro>
-          <Box pad={{ horizontal: calcPad(size) }} width={{ max: '900px' }}>
+          <Box width={{ max: '700px' }}>
```
Comment (halocline) [inline aries-site/src/pages/index.js:86]: Can we add a comment as to how this value was arrived at?

Also, the `xxxsmall` value below looks suspect. I don't believe we have a value for that.

**Comment by taysea** [comment]
@jcfilben @halocline I have been using [v2](https://github.com/grommet/hpe-design-system/pull/2433) for cleanups and the direction to use theme as is. I've incorporated all the comments here in the new PR.

**Pull request closed without merging**